style.css

```diff
@@ -312,6 +331,82 @@ form > textarea {
   font-size: 1.2em;
 }
 
+/* Dynamic Project Cards */
+.project_overlay {
+  position: fixed;
+  margin: auto;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 2;
+  cursor: pointer;
+  border-radius: 10px;
+  background-color: #c1c7d0;
+}
```

Comment:
- [ ] There are some things that could be improved regarding the style, remember that one of the main goals of building this portfolio project is to stick as much as possible to a given design 🎨. Please, consider the following changes in your popup window:

  - [ ] 🚧 Style the close button so it matches the template. 
  - [ ] 🚧 Style the action buttons (See live and See source) so they match the template.
  - [ ] 🚧 Consider giving some rounds to your popup borders.

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/202532688-b64452bf-9fb7-4ebe-9dcf-73f25af157d0.png) | ![imageedit_8_8567744313](https://user-images.githubusercontent.com/77069011/202532535-3209ed60-7bbd-423a-888c-e207e607251e.gif) |

Please, fix those, you're almost there! 🤓 🛠️